ansible/roles/infra-osp-resources-destroy/tasks/main.yml

```diff
@@ -1,2 +1,15 @@
 ---
-- import_tasks: delete_osp_resources.yml
\ No newline at end of file
+- import_tasks: detect_project.yml
+
+- when: osp_project_info | length > 0
```

Comment:
As mentioned in other comment, heat stack deletion needs to come after `project.yml` or the heat stack will sometimes fail to delete due to dependencies created outside heat stack.